### PR TITLE
Houdini: Extend op menu to create HDAs in a Houdinish way

### PR DESCRIPTION
**Pull request by MustafaJafar**
## Changelog Description
Add  `Create New (AYON)...`  to OPMenu to allow artists to create AYON HDA product in a Houdini native way.

Animation_59

## Additional info
This enhancement was mentioned by @fabiaserra  in https://github.com/ynput/ayon-core/pull/456#issuecomment-2096575818 and other clients.

## Additional info 2
Currently, when RMB on a subnetwork, it'll be moved to a new subnetwork.
There's another PR https://github.com/ynput/ayon-core/pull/456 that enhances HDA creator plugin selection. 
when using both of the PRs, the HDA creator plugin will convert the subnetwork directly without moving it to another subnetwork.

## Testing notes:
1. Create nodes in OBJs in Houdini
2. Right Button Click the node and select  `Digital Assets` > `Create New (AYON)...`
3. Publish

### CODE REVIEW

File: client/ayon_core/hosts/houdini/startup/OPmenu.xml [hunk 1]
```diff
@@ -0,0 +1,543 @@
...
+            </context>
+            <scriptCode><![CDATA[
+from houdinihelp import examples
+examples.save_node_example(kwargs['node'])
+]]></scriptCode>
+        </scriptItem>
+        -->
+
+        <!-- Node flags submenu. -->
+        <subMenu id="opmenu.flags_menu">
```
Comment (fabiaserra) [inline client/ayon_core/hosts/houdini/startup/OPmenu.xml:76]: where is all of these coming from?

Comment (fabiaserra) [reply client/ayon_core/hosts/houdini/startup/OPmenu.xml:76]: correct

Comment (fabiaserra) [reply client/ayon_core/hosts/houdini/startup/OPmenu.xml:76]: as explained here https://www.sidefx.com/docs/houdini/basics/config_menus.html#reorganizing-menus

Comment (MustafaJafar) [reply client/ayon_core/hosts/houdini/startup/OPmenu.xml:76]: oh, I think I missed! 
Let me try again.

Comment (MustafaJafar) [reply client/ayon_core/hosts/houdini/startup/OPmenu.xml]: Please let me know what do you think
https://github.com/ynput/ayon-core/pull/504/commits/86b2f3b5b52431f6cb11b91e771552c717fa6df8

Comment (fabiaserra) [reply client/ayon_core/hosts/houdini/startup/OPmenu.xml]: much better yeah, ty!

Comment (BigRoy) [reply client/ayon_core/hosts/houdini/startup/OPmenu.xml]: definitely much better!

**Review by BigRoy — APPROVED** [review]
**Didn't test the code** but visually I like how this works in the demo video.

**Review by moonyuet — APPROVED** [review]

**Comment by antirotor** [comment]
Is https://github.com/ynput/ayon-core/pull/456 requirement for this to be merged?

**Comment by MustafaJafar** [comment]
> Is https://github.com/ynput/ayon-core/pull/456 requirement for this to be merged?

No, they are independent.

However, #456 enhances the selection.

**Review by antirotor — APPROVED** [review]
